vyper/venom/passes/sccp/sccp.py

```diff
@@ -169,7 +170,8 @@ def _visit_expr(self, inst: IRInstruction):
             target = self.fn.get_basic_block(inst.operands[0].value)
             self.work_list.append(FlowWorkItem(inst.parent, target))
         elif opcode == "jnz":
-            lat = self.lattice[inst.operands[0]]
+            lat = _get_lattice(self.lattice, inst.operands[0])
```

Comment:
I believe the `_get_lattice` name for the function is confusing, as what it actually does is either returns the actual lattice or if you are looking up a literal (which shouldn't happen) it returns that.

Reply:
there are other places in the file which guard lattice lookups with an `isinstance(..., IRVariable)` check